packages/calling/calling/src/Contacts/ContactsClient.ts

```diff
@@ -52,160 +77,255 @@ export class ContactsClient implements IContacts {
   }
 
   /**
-   * @param contact - Encrypted contact object.
-   * @param avatarUrlDomain - Avatar Url.
-   * @param encryptionKeyUrl - Encryption key received from KMS.
-   * @param ownerId - OwnerId.
-   * @returns Promise for decrypted contact object.
+   * Decrypt emails, phoneNumbers, sipAddresses.
+   *
+   * @param contactDetails - Array ContactDetail.
+   * @param encryptionKeyUrl - KMS encryptionKeyUrl for the contact.
+   * @returns Decrypted phoneNumbers.
    */
-  private async decryptContact(
-    contact: Contact,
-    avatarUrlDomain: string,
+  private async decryptContactDetail(
```

Comment:
nice to have it on the global name space we could reuse in other places if needed